### PR TITLE
Move plugin tutorial to team ownership

### PR DESCRIPTION
**Pull request by SkySkimmer**
The team isn't created yet, I guess we could start with people who have commit access to the original tutorial repo (@ybertot @ejgallego @gares @mattam82 @ppedrot @SkySkimmer)

### CODE REVIEW

**Review by ejgallego — APPROVED** [review]

File: .github/CODEOWNERS
Comment (gares) [inline .github/CODEOWNERS]: is `rial` such a big deal? :-)

**Review by gares — APPROVED** [review]

**Comment by ejgallego** [comment]
I'm taking this; I don't see any argument against so I will merge in a couple of days if nobody opposes.

At some point the team should become more like "api-maintainers", but this is a good first step for now IMHO.

**Comment by SkySkimmer** [comment]
Don't forget to create the team when you merge.

**Pull request merged**
